mlir/lib/Dialect/LLVMIR/IR/LLVMDialect.cpp

```diff
@@ -1262,6 +1325,18 @@ static ParseResult parseCallTypeAndResolveOperands(
   if (funcType.getNumResults() != 0)
     result.addTypes(funcType.getResults());
 
+  if (!isVarArg) {
```

Comment:
It is required if the callee is vararg, however, it can still be present for a non-vararg call, and we do not need (or want) to print it if the callee is non-vararg as the type is the same as the mlir op type printed after the `:`.

Reply:
Oh you are indeed correct, we do not need to set that.